persistence/sql/migrations/gomigrations/identity_pk.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package gomigrations
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+
+	"github.com/gobuffalo/pop/v6"
+	"github.com/pkg/errors"
+
+	"github.com/ory/x/popx"
+)
+
+func path() string {
+	_, file, line, _ := runtime.Caller(1)
+	return fmt.Sprintf("%s:%d", filepath.Base(file), line)
+}
+
+var ChangeAddressesPK = []popx.Migration{
+	{
+		Version:   "20241001000000000000",
+		Path:      path(),
+		Name:      "Change primary key for identity_verifiable_addresses",
+		Direction: "up",
+		Type:      "go",
+		DBType:    "cockroach",
+		RunnerNoTx: func(m popx.Migration, c *pop.Connection) error {
+			_, err := c.Store.Exec("ALTER TABLE identity_verifiable_addresses ALTER PRIMARY KEY USING COLUMNS (identity_id,id)")
```

Comment:
Move to SQL files since we don't need go for these?

Reply:
Currently this is the only way to run crdb migrations outside of transactions

Reply:
I see!